pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, mkDerivation, fetchFromGitHub
+, cmake, libressl, SDL2, qtbase, python2, alsaLib
+, useVulkan ? true, vulkan-loader, vulkan-headers
+}:
+
+mkDerivation rec {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  src = fetchFromGitHub {
+    owner = "yuzu-emu";
+    repo = "${pname}";
+    fetchSubmodules = true;
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s9p5di9kcr8sh6iipv21b74c4dhn37an4khy4y7b9s9y8p1haim";
+  };
+
+  nativeBuildInputs = [ cmake libressl ];
```

Comment:
Unless libressl is required at build time, it should go to build inputs.

Reply:
This has been discussed before, this is a special case.